src/library/scala/collection/mutable/AnyRefMap.scala

```diff
@@ -293,6 +293,7 @@ class AnyRefMap[K <: AnyRef, V] private[collection] (defaultEntry: K => V, initi
   }
 
   /** Adds a new key/value pair to this map and returns the map. */
+  @deprecated("Use `addOne` or `update` instead; infix operations with an operand of multiple args will be deprecated", "2.13.3")
```

Comment:
I would just mention update and not addOne, since they're equivalent except the first one will box.

Reply:
There is specialized `addOne`. Or was that only `LongMap`? I'll doubly check.

Reply:
```
  /** Adds a new key/value pair to this map and returns the map. */
  @inline final def addOne(key: K, value: V): this.type = { update(key, value); this }

  @inline override final def addOne(kv: (K, V)): this.type = { update(kv._1, kv._2); this }
```

Reply:
There is the caveat about which interface is in use, but we know they're selecting from the subtype for `+=`.

Reply:
This would be a good place to say `"Use adderAll instead"`.